### PR TITLE
Fix Travis CI config

### PR DESCRIPTION
**Pull request by tspencer244**
Fixes PHPMailer/PHPMailer#2168

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,7 +62,3 @@ jobs:
     - php: 7.4
       env: CODE_COVERAGE=1
     - php: nightly
-
-matrix:
```
Comment (tspencer244) [inline .travis.yml:62]: `matrix` is an alias of `jobs` so I have just merged these keys

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 language: php
+dist: xenial
+os: linux
```
Comment (tspencer244) [inline .travis.yml:3]: These root keys were defaulting to the configured values so I have just added them here to address all configuration validation issues reported by Travis CI

**Comment by Synchro** [comment]
Ah, I didn't know matrix and jobs were the same thing. Does this fix the coding standards stage?

**Comment by tspencer244** [comment]
> Ah, I didn't know matrix and jobs were the same thing. Does this fix the coding standards stage?

I believe it does - I can see the coding standards stage being run before the tests, and the tests wait for coding standards to pass first

**Comment by Synchro** [comment]
Thank you!

**Pull request merged**